tests/integrationtests/tst_perfparser.cpp

```diff
@@ -172,6 +172,107 @@ private slots:
         m_capabilities = host.perfCapabilities();
     }
 
+    void testFileErrorHandling()
+    {
+        PerfParser parser(this);
+        QSignalSpy parsingFailedSpy(&parser, &PerfParser::parsingFailed);
+        QString message;
+
+        const auto notThereFile = QLatin1String("not_here");
+        parser.initParserArgs(notThereFile);
+
+        COMPARE_OR_THROW(parsingFailedSpy.count(), 1);
```

Comment:
just `QCOMPARE` - the `COMPARE_OR_THROW` is only needed in helper functions that don't return `void`

Reply:
both `QCOMPARE` and `QVERIFY` were taken from the other test codes in the exact same file... but I'll change all of them in the same commit, then...